tests/shared/robot.spec.js

```diff
@@ -131,16 +131,16 @@ describe('Shared: Robot', function() {
         data: {
           message: 'foo',
           code: 'MYERROR',
-          stacktrace: error.stack
+          stack: error.stack
         }
       });
     });
 
     it('should handle a string error', function() {
```

Comment:
(`robot.printError` by comparison doesn't use `.getTitle` and `.getDescription` - it just bubbles down to `errors.toJSON`)